pkg/sunmao/core.go

```diff
@@ -1,8 +1,12 @@
 package sunmao
 
+type TypeMeta struct {
```

Comment:
`VK` or `VersionAndKind` or `VersionKind` may be more straightforward to understand, especially since we already have the `Metadata` field.

(Similiar to k8s's gvk)